src/components/Asset/AssetContent/EditHistory.tsx

```diff
@@ -10,20 +10,38 @@ import styles from './EditHistory.module.css'
 const getReceipts = gql`
   query ReceiptData($address: ID!) {
     nftUpdates(
-      where: { id: $address }
+      where: { nft: $address }
       orderBy: timestamp
       orderDirection: desc
     ) {
       id
+      nft {
+        address
+      }
       tx
       timestamp
+      type
     }
   }
 `
 
 export default function EditHistory(): ReactElement {
   const { ddo } = useAsset()
 
+  function getUpdateType(type: string): string {
+    switch (type) {
+      case 'METADATA_CREATED':
+        return 'published'
+      case 'METADATA_UPDATED':
+        return 'updated'
+      case 'STATE_UPDATED':
+        return 'state updated'
+      case 'TOKEN_UPDATED':
```

Comment:
this is `TOKENURI_UPDATED`, not token. It's when `tokenUri` is updated on the nft